src/LiveDevelopment/LiveDevMultiBrowser.js

```diff
@@ -91,8 +92,8 @@ define(function (require, exports, module) {
 
     // XXXBramble: we need to get this loaded so our extension can require/use it later
     require("LiveDevelopment/Servers/BaseServer");
-    
```

Comment:
These types of changes are a bad idea, because they a) aren't related to your actual change; b) make the history of this file (i.e., blame) look like you touched them (when really you didn't).  Probably your editor did this for you?  I'd set your editor to not randomly change whitespace in files for lines you don't touch (those you do touch, this is a good fix).

Reply:
@humphd inside of LiveDevMultiBrowser.js I am seeing these icons in my atom ide. I'm not sure if this is related to the change made at line 94.

![livedevmultibrowser_changes](https://user-images.githubusercontent.com/15128090/33586981-01bd36cc-d93a-11e7-9c02-6894a7807c23.JPG)

Reply:
If you want me to look at this with you tomorrow, I'd be happy to help you solve it.  It looks like you've got a merge/rebase conflict, which we can sort out and tell git what to take.

Reply:
That would be great if we could. I appreciate it.